src/shared/palette.ts

```diff
@@ -85,3 +85,7 @@ export const ERROR = {
   main: '#DD0E0E',
   light: '#FFC799',
 };
+
+export const WHITE = {
+  main: '#FFFFFF',
+};
```

Comment:
I think it's a bit strange since it would be the only color we use that it's not handled by the theming.

And isn't it this one from the figma design?

![test](https://user-images.githubusercontent.com/1252707/207619843-209efabf-1537-4c33-b690-bb82e3867f33.PNG)